gobblin-core/src/main/java/gobblin/source/extractor/extract/kafka/ConfluentKafkaSchemaRegistry.java

```diff
@@ -48,6 +48,11 @@
   public static final String CONFLUENT_MAX_SCHEMAS_PER_SUBJECT =
       "kafka.schema_registry.confluent.max_schemas_per_subject";
 
+  public static final String CONFLUENT_SCHEMA_NAME_SUFFIX = "kafka.schema_registry.confluent.schema.name.suffix";
```

Comment:
Can you change it to: kafka.schema_registry.confluent.schema_name_suffix  to be consistent in style with the other config keys?